package com.kpv.bankcardsmanagement.domain.core.model

data class AuthTokens(
    val accessToken: String,
    val refreshToken: String
)